sample/rules.go

```diff
@@ -233,6 +234,13 @@ func extractValueFromSpan(trace *types.Trace, span *types.Span, condition *confi
 	var value any
 	var exists bool
 	for _, field := range condition.Fields {
+		// check if field prefix "root."
+		if strings.HasPrefix(field, "root.") {
```

Comment:
let's pull this prefix out into a constant, and when we strip it off on the next line, strip by len(constant). That way we're safe if we decide to change the name.

Reply:
Updated this!